### PR TITLE
Add description for modules in section `seealso`

### PR DESCRIPTION
**Pull request by bmillemathias**
##### SUMMARY
Add description for modules in section `seealso`.
Of course that would be better if the code could use the value of `short_description` of the target module.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docsite

### CODE REVIEW

**Comment by bmillemathias** [comment]
The PR could also be backported to `2.11`

File: lib/ansible/modules/import_role.py [hunk 1]
```diff
@@ -86,9 +86,15 @@
   - Unlike M(ansible.builtin.include_role) variable exposure is not configurable, and will always be exposed.
 seealso:
 - module: ansible.builtin.import_playbook
+  description: Import a playbook.
+- module: ansible.builtin.import_role
+  description: Import a role into a play.
```
Comment (felixfontein) [inline lib/ansible/modules/import_role.py]:
```suggestion
```

File: lib/ansible/modules/service.py [hunk 2]
```diff
@@ -89,7 +89,12 @@
     - For AIX, group subsystem names can be used.
     - Supports C(check_mode).
 seealso:
-    - module: ansible.windows.win_service
+- module: ansible.builtin.sysvinit
+  description: Manage SysV services.
+- module: ansible.builtin.service
+  description: Generic module to manage a service.
```
Comment (felixfontein) [inline lib/ansible/modules/service.py]:
```suggestion
```
You probably want to reference `ansible.builtin.systemd` instead.

File: lib/ansible/modules/import_role.py (hunk 1 above)
Comment (bmillemathias) [reply lib/ansible/modules/import_role.py]: thanks for spotting the mistake

File: lib/ansible/modules/service.py (hunk 2 above)
Comment (bmillemathias) [reply lib/ansible/modules/service.py]: indeed !

**Comment by bcoca** [comment]
Hi!

Thanks very much for your submission to Ansible.  It sincerely means a lot to us that you've taken time to contribute.

Unfortunately, we are going to decline this PR because this requires maintaining data 'short description' in multiple places for a module, it makes more sense to update the doc presentation layer to fetch this information and present it than to copy it all over.

However, we're absolutely always up for discussion.  
Since this is a really busy project, we don't always see comments on closed tickets, but want to encourage open dialog.  

You can stop by the development list, and we'd be glad to talk about it - and we might even be persuaded otherwise!

   * https://groups.google.com/forum/#!forum/ansible-devel

In the future, sometimes starting a discussion on the development list prior to implenting a feature can make getting things included a little easier, but it's not always necessary.

Thank you once again for this and your interest in Ansible!

**Pull request closed without merging**